sdk/src/loom/ModestMaps/core/Tile.ls

```diff
@@ -208,7 +215,7 @@ package loom.modestmaps.core
 
             //dispose all child Images
             while (numChildren > 0) {
```

Comment:
There's `removeChildren`, but we also want to add them back to the pool. I don't think tiles have a lot of images (or even above one), so I don't think this is a bottleneck.